critiquebrainz/frontend/external/musicbrainz_db/tests/release_group_test.py

```diff
@@ -78,3 +79,19 @@ def test_fetch_browse_release_groups(self):
             }
         ])
         self.assertEqual(release_groups[1], 2)
+
+    def test_unknown_release_group(self):
+        mb_release_group.get_entities_by_gids = MagicMock()
+        mb_release_group.get_entities_by_gids.return_value = {
```

Comment:
If we mock the result of `get_entities_by_gid`, do any of the changes in the function even get tested?

Reply:
For improving this, I've mocked the result of the `mbdata` queries. 
I've also mocked the `reviewed_entities` function (I think it is okay to do so, since we now have a test for it in `db/review_test.py`). 
The test now handles the changes made to the function `get_entities_by_gid` as well as tests all the serialization stuff as before.